src/aspire/basis/__init__.py

```diff
@@ -2,6 +2,24 @@
 # isort: off
 
 from .basis import Basis, Coef, ComplexCoef
+from .basis_utils import (
```

Comment:
These are all internal utility functions, from what I can tell. Why do we want to expose them as part of the public API?

Reply:
Yeah, you are right, I'm not sure what I was thinking there.  I will revert these. Sorry!

Reply:
Thanks!